### DISCUSSION TITLE
Bind `consult-line` to `C-f` does not work in init, but works manually

### DISCUSSION THREAD

**Discussion by Sinthoras7** [minad/consult#1099] [Q&A]
Thank you for developing this wonderful package!

I have a question regarding the binding of `consult-line` and I am quite sure it is not the fault of your package, but I don't know where else to ask. If I bind the command to `C-s` via the following code everything works as expected.
```
    (use-package consult
         :bind
         ("C-s" . consult-line)
      )
```
But if I bind it to `C-f` instead, it does not work. I thought it may be because the key is already bound, but even a ` (global-unset-key (kbd "C-f"))` before it does not help. 

Strangely enough if I execute the commands manually it works, just not if they are in the init. (I do not change the keybinding of `C-f` after that. At least not that I am aware of, though evil mode may do some strange stuff.)

My greater goal is to use the more standard keybindings `C-s` for saving a buffer and `C-f` for searching, as I came into in a situation where I need to save often and the two keys `C-x C-s` are becoming annoying.

**Comment by minad** [comment]
> My greater goal is to use the more standard keybindings C-s for saving a buffer and C-f for searching, as I came into in a situation where I need to save often and the two keys C-x C-s are becoming annoying.

Such general key binding questions seem out of scope this discussion forum. The Consult commands behave like every other commands, so the reference manual for use-package and regarding key bindings applies.
